app/controllers/api/v2/application_controller.rb

```diff
@@ -1,6 +1,8 @@
 module API
   module V2
     class ApplicationController < ::ApplicationController
+      attr_reader :current_user
```

Comment:
I wonder if we could use the new current attributes from rails 5.2 here?

Reply:
I've not come across this yet – anything you can point me towards?

Reply:
Create a class or concern like so -

```
class Current < ActiveSupport::CurrentAttributes
  attribute :user
end
```

then you can assign `Current.user = @user` anywhere which gives you access to `current_user` throughout the app.

Haven't used it myself, just read about it.

Reply:
Don't let this stop you merging...